### ISSUE TITLE
Adding licences list when importing an image

### ISSUE THREAD

**Issue by Anthere** [openzim/wikifundi#56]
As is http://wikifundi-fr.openzim.org/wiki/Fichier:Car-pooling.jpg

The list is available in the preferences

http://wikifundi-fr.openzim.org/wiki/Spécial:Préférences#mw-prefsection-uploads
but is not available in the dropdown manu when uploading pictures

**Comment by florentk** [comment]
@Anthere @kelson42 I think we must add "MediaWiki:Licenses" in mirroring list pages

**Comment by Anthere** [comment]
We have a problem. 
The list of proposed licences is the very restricted one proposed on Wikipedia. 
It should be the list proposed on Wikimedia Commons

**Comment by florentk** [comment]
@Anthere I see, but I can not take that in Wikimedia Common, but we can create the MediaWiki: Licenses page in the Wikifundi content to create our custom version

**Comment by Anthere** [comment]
Let's do that ! I'll be happy with a set of 4-6 most used licenses/PD to get the ball rolling, plus the option "I have no idea what a licence is" or "I do not know what the the licence of this media is" type of approach.

**Comment by Anthere** [comment]
I could not fix that. I search for a MediaWiki:Licenses page or MediaWiki:Licenses pages on Wikipedia but could not find anything under that name. I search on MediaWiki but did not find those pages either. Any better pointer ? If not, you will have to fix that yourself...

**Comment by florentk** [comment]
I use this page for moment : 
https://fr.wikipedia.org/wiki/MediaWiki:Licenses
https://en.wikipedia.org/wiki/MediaWiki:Licenses

The idea is put a new version of theses pages in Wikifundi project

**Comment by florentk** [comment]
@Anthere I see that for FR version, I don't found the page also, this is very strange ...

**Comment by florentk** [comment]
@Anthere I thinks that is this page : https://commons.wikimedia.org/wiki/MediaWiki:Licenses/fr
You can copy content in https://fr.wikipedia.org/wiki/Projet:WikiFundi_Contenu/MediaWiki:Licences ?

**Comment by Anthere** [comment]
French : https://fr.wikipedia.org/wiki/Projet:WikiFundi_Contenu/MediaWiki:Licences

English : https://en.wikipedia.org/wiki/Wikipedia:WikiProject_WikiFundi_Content/MediaWiki:Licenses

**Comment by Anthere** [comment]
Florent, I tried to change the content of the English licences (which I put at http://wikifundi-en.openzim.org/wiki/MediaWiki:Licences); It does not make any difference so I am obviously not using the right "url". Where should the English licences be put ?

**Comment by florentk** [comment]
@Anthere oups, little language subtlety, the good page is http://wikifundi-en.openzim.org/wiki/MediaWiki:Licen**S**es

**Comment by Anthere** [comment]
gnnnnn.
Stupid me. 
Need more coffee...

**Comment by Anthere** [comment]
Modifications made on 9th of July not integrated into current version (based on 8th of July). Final version needs to pick up a fresh version of the WikiFundi content on Wikipedia.

**Comment by Anthere** [comment]
Both languages. 
None of the pages have been imported (in both languages, they have been created on the 9th, AFTER the archive was done). 
For example, in English : https://en.wikipedia.org/wiki/Wikipedia:WikiProject_WikiFundi_Content/MediaWiki:Licenses should be the right version. But it is still the old version you made. 
Also, all pages that are templates of the licences are missing, such as 
https://en.wikipedia.org/wiki/Wikipedia:WikiProject_WikiFundi_Content/Template:Creative_Commons_Cc-zero
https://en.wikipedia.org/wiki/Wikipedia:WikiProject_WikiFundi_Content/Template:Public_domain
https://en.wikipedia.org/wiki/Wikipedia:WikiProject_WikiFundi_Content/Template:Creative_Commons_cc-by-sa-4.0
https://en.wikipedia.org/wiki/Wikipedia:WikiProject_WikiFundi_Content/Template:Creative_Commons_cc-by-sa-3.0
https://en.wikipedia.org/wiki/Wikipedia:WikiProject_WikiFundi_Content/Template:Creative_Commons_cc-by-3.0
https://en.wikipedia.org/wiki/Wikipedia:WikiProject_WikiFundi_Content/Template:Unknown_licence

Same for French version. The pages were made on Wikipedia FR and EN. But the archive being older than all those changes, the pages were not imported, hence licenses not working.

**Comment by florentk** [comment]
I see any difference between http://wikifundi-en.openzim.org/wiki/MediaWiki:Licenses and https://en.wikipedia.org/wiki/Wikipedia:WikiProject_WikiFundi_Content/MediaWiki:Licenses

**Comment by Anthere** [comment]
It was because the content of the sim was not from the bot. It was updated by myself when I was testing.

**Comment by kelson42** [comment]
@Anthere I'm not sure I fully understand the problem, but can you please point out the exact list of articles missing on http://wikifundi-en.openzim.org/ and http://wikifundi-fr.openzim.org/ ? Like @florentk said, things seem properly migrated!

**Comment by Anthere** [comment]
The problem existed because the content had been migrated on the 8th of July whilst the pages had been created on the 9th of July. So obviously, it was not the right content. I just checked. The pages have been properly imported on the openzim in French and English. I tried an import and everything was under order. The right page has been uploaded on the 19th of September...
So crossing fingers, this issue can now be closed.
